imqsauth/http.go

```diff
@@ -776,6 +798,33 @@ func httpHandlerUpdateUser(central *ImqsCentral, w http.ResponseWriter, r *httpR
 	}
 }
 
+func httpHandlerUnlockUser(central *ImqsCentral, w http.ResponseWriter, r *httpRequest) {
+	username := strings.TrimSpace(r.http.URL.Query().Get("username"))
+
+	userId, err := getUserId(r)
+	if err != nil {
+		authaus.HttpSendTxt(w, http.StatusBadRequest, err.Error())
+		return
+	}
+
+	// Get the userId of the logged-in user
+	user := authaus.AuthUser{
+		UserId:   authaus.UserId(userId),
+		Username: username,
+	}
+
+	x := central.Central
+	clientIPAddress := getIPAddress(r.http)
+
+	if err := x.UnlockAccount(userId); err != nil {
```

Comment:
I think UnlockAccount should call Stats.ResetInvalidPasswordHistory. That seems like an authaus implementation detail that the user (ie imqsauth) shouldn't have to remember.